terraform/state-persistence.tf

```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "gt-site-cluster" {
```

Comment:
Remote state for a given infrastructure is supposed to be pushed and pulled during each deployment via the [`remote`](https://www.terraform.io/docs/commands/remote.html) subcommand of `terraform`.

The `terraform_remote_state` data source is setup to be a read-only data source for scenarios where you're trying to extract references to resources created by someone else's Terraform driven infrastructure:

An example from the Terraform docs:

> For example, a core infrastructure team can handle building the core machines, networking, etc. and can expose some information to other teams to run their own infrastructure. As a more specific example with AWS: you can expose things such as VPC IDs, subnets, NAT instance IDs, etc. through remote state and have other Terraform states consume that.